utils/scripts/gen_tflite_models.py

```diff
@@ -116,6 +122,14 @@ def save_model(model, filename):
         output_arrays=model_outputs_array,
     )
     converter.dump_graphviz_video = False
+    
+    if (quant):
+      converter.optimizations = [tf.lite.Optimize.DEFAULT]
+      converter.representative_dataset = representative_dataset
```

Comment:
We should provide this through the `save_model` function API and not through the global variable. Also for multiple model inputs should this be an array?

Reply:
Agree, there is now a generator that is passed with save_model and also, the generator is made generic - it simply accepts the list of any length. Verifed with unary and binary models only.